tasks/settings.yml

```diff
@@ -17,6 +17,13 @@
     line: 'JENKINS_HOME={{ jenkins_home }}'
   register: jenkins_home_config
 
+- name: Set the user and group that Jenkins runs as
```

Comment:
When I add this to `defaults/main.yml`:

```
jenkins_init_changes:                                                           
  - option: "JENKINS_ARGS"                                                      
    value: "--prefix={{ jenkins_url_prefix }}"                                  
  - option: "{{ jenkins_java_options_env_var }}"                                
    value: "{{ jenkins_java_options }}"                                         
  - option: "JENKINS_USER"                                                      
    value: "{{ jenkins_process_user }}" 
```
I end up with:

```
JENKINS_USER=jenkins
JENKINS_USER="$JENKINS_USER jenkins"
```
and get this task error

```
fatal: [jenkins]: FAILED! => {"changed": false, "msg": "Unable to restart service jenkins: Job for jenkins.service failed because the control process exited with error code. See \"systemctl status jenkins.service\" and \"journalctl -xe\" for details.\n"}
```

Reply:
@geerlingguy your suggested changes doesn't work because it produces a list of values rather than the single value that the key requires.